tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -50,7 +51,7 @@ def restore(self, keys) -> dict:
         vals = self._merge_items(first_items, remaining_items)
         return vals
 
-    def chunk_keys(self, keys: list) -> list:
+    def chunk_keys(self, keys: List[dict]) -> List[List[dict]]:
```

Comment:
non-blocking suggestion:
if we wanted to get more advanced, i think we can do some typing shenanigans so that this looks something like
```
def chunk_keys(self, keys: List[dict[K, V]]) -> List[List[dict[K, V]]]:
```
or honestly, we could be a little more generic here with something like
```
def chunk_keys(self, keys: Sequence[T]) -> List[List[T]]:
```
since this should work even with a list/tuple/etc

Reply:
Will do
```
chunk_keys(self, keys: Sequence[T]) -> List[Sequence[T]]
```
cause if i do list[list[t]], I get 
```
tron/serialize/runstate/dynamodb_state_store.py: note: In member "chunk_keys" of class "DynamoDBStateStore":
tron/serialize/runstate/dynamodb_state_store.py:64:16: error: Incompatible return value type (got "List[Sequence[T]]", expected "List[List[T]]")  [return-value]
            return cand_keys_chunks
```